jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/ProcessInstanceDiagramPresenter.java

```diff
@@ -162,34 +164,42 @@ protected void loadProcessInstanceDetails() {
 
             view.setParentSelectedCommand(() -> listenParentProcessInstanceSelected());
 
-            timerInstances = summary.getTimerInstances().stream().sorted(comparing(TimerInstanceSummary::getName, String.CASE_INSENSITIVE_ORDER).thenComparingLong(TimerInstanceSummary::getId)).collect(toList());
-
-            timerInstances.forEach(ti -> {
-                ti.setDescription(constants.NextExecution() + " " + DateUtils.getPrettyTime(ti.getNextFireTime()));
-                ti.addCallback(constants.Reschedule(),
-                               () -> {
-                                   rescheduleView.setOnReschedule(timer -> onTimerInstanceReschedule(timer));
-                                   rescheduleView.setValue(ti);
-                                   rescheduleView.show();
-                               });
-            });
-
-            view.setTimerInstances(timerInstances);
-
             view.setShowOrHideNodeActionsCommand(() -> showOrHideNodeActions(forLog, processInstance));
 
             timers = summary.getProcessDefinition().getTimers().stream().collect(toList());
 
             view.showOrHideNodeActionsTriggered();
             expandDiagram(processInstance, summary);
 
-            view.onShow();
         }).getProcessInstanceDiagramSummary(processInstance.getProcessInstanceKey(),
                                             preferences.getProcessInstanceDiagramCompletedNodeColor(),
                                             preferences.getProcessInstanceDiagramCompletedNodeBorderColor(),
                                             preferences.getProcessInstanceDiagramActiveNodeBorderColor());
     }
 
+    protected void loadTimerInstances() {
+        if (processInstance.getState() == 1) {
```

Comment:
do we have the state as a constant somewhere? Would be nice to replace the literal here.

Reply:
you are right. Done